lms/envs/common.py

```diff
@@ -413,7 +413,7 @@
     'ENABLE_OPENBADGES': False,
 
     # Credit course API
-    'ENABLE_CREDIT_API': False,
+    'ENABLE_CREDIT_API': True,
```

Comment:
There is no compelling reason to feature-flag this API. If maintainers don't want to use it...they simply do nothing.

Note that this is a merging of four commits that were previously intended to go out with a regular release.